controllers/kubeletconfig_controller_test.go

```diff
@@ -129,6 +129,22 @@ var _ = Describe("Test KubeletConfig Reconcile", func() {
 				event := <-fakeRecorder.Events
 				Expect(event).To(ContainSubstring("ProcessFailed"))
 			})
+
+			It("should skip invalid kubeletconfig", func() {
+				invalidMcoKc := testobjs.NewKubeletConfigWithoutData("test1", label1, mcp1.Spec.MachineConfigSelector)
+				reconciler, err := NewFakeKubeletConfigReconciler(nro, mcp1, invalidMcoKc)
+				Expect(err).ToNot(HaveOccurred())
+
+				key := client.ObjectKeyFromObject(mcoKc1)
+				_, err = reconciler.Reconcile(context.TODO(), reconcile.Request{NamespacedName: key})
+				Expect(err).ToNot(HaveOccurred())
+
+				// verify creation event
+				fakeRecorder, ok := reconciler.Recorder.(*record.FakeRecorder)
+				Expect(ok).To(BeTrue())
+				event := <-fakeRecorder.Events
+				Expect(event).To(ContainSubstring("ProcessSkip"))
```

Comment:
"ProcessSkip"  is a constant ?  should it be ?   (I'm trying to think how to link the desired test string here to the code if at all)

Reply:
we don't have a formal API contract about these values (so yes, the API is leaking)